src/ukify/ukify.py

```diff
@@ -278,6 +292,27 @@ def parse_arg(cls, s):
 
         return cls.create(name, contents)
 
+    @classmethod
+    def parse_text_arg(cls, s):
+        try:
+            name, ttype = s.split(':')
+        except ValueError as e:
+            raise ValueError(f'Cannot parse section spec (name or type missing): {s!r}') from e
+
+        if '@' in ttype:
+            try:
+                ttype, path = ttype.split('@')
+            except ValueError as e:
+                raise ValueError(f'Cannot parse section spec (type or path missing): {ttype!r}') from e
```

Comment:
Now this message is wrong. It will not fail if type of path is missing, because in that case this block would be skipped. It will fail if there's an additional `@`, i.e. if there's too much stuff, not too little.